src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,19 @@
+import { context, getOctokit } from "@actions/github";
+import { setFailed } from "@actions/core";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+export const  requireCC0 = () => {
+  const github = getOctokit(GITHUB_TOKEN).rest;
+  const copyrightRegex = /^[\s\S]*\n## Copyright\nCopyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g;
```

Comment:
```suggestion
  const copyrightRegex = /^[\s\S]*\n## Copyright\n{1,2}Copyright and related rights waived via \[CC0\]\(\.\.\/LICENSE.md\)\.\n?$/g;
```
Some people put a blank line below headings.  Would be nice if we could support that style.